File: Laboratorio_04/Python/Calculadora/CalculatorImpl.py
# main.py

import Calculator

def main():
    a = 10
    b = 5

    print("Addition:", Calculator.add(a, b))
    print("Subtraction:", Calculator.sub(a, b))
    print("Multiplication:", Calculator.mul(a, b))
    print("Division:", Calculator.divide(a, b))

    b = 0
    print("Division with zero:", Calculator.divide(a, b))

if __name__ == "__main__":
    main()

File: Laboratorio_04/c/calculadora/CalculatorImpl.c
#include "Calculator.h"
int add(int a, int b) {
    return a + b;
}
int sub(int a, int b) {
    return a - b;
}
int mul(int a, int b) {
    return a * b;
}
int divide(int a, int b) { 
    if (b == 0) {
        return 0; // Manejar división por cero
    }
    return a / b;
}